code/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2020.1.2"
```

Comment:
If we change the build version on master, all of the driver stations will need to download the new version the first time it builds.

Reply:
Possibly. Though we should probably all be updating to the latest builds. Better to know if something doesn't work sooner than later :) We can always revert back.

I'll merge with this setting, but if it causes issues can rollback. Its a small change locally to resolve the issue.